packages/mask/background/database/avatar-cache/avatar.ts

```diff
@@ -1,6 +1,6 @@
 import { ProfileIdentifier } from '@masknet/shared-base'
 import { queryAvatarDB, isAvatarOutdatedDB, storeAvatarDB, IdentifierWithAvatar, createAvatarDBAccess } from './db'
-import { MaskMessages } from '../../../shared'
+import { MaskMessages } from '@masknet/plugin-wallet'
```

Comment:
Move this migration to another PR.

Reply:
don't do this migration. it should not be moved to '@masknet/plugin-wallet'